<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ColorSelect</class>
 <widget class="QWidget" name="ColorSelect">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>328</width>
    <height>241</height>
   </rect>
  </property>
  <property name="baseSize">
   <size>
    <width>0</width>
    <height>0</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>ColorSelect</string>
  </property>
  <property name="autoFillBackground">
   <bool>true</bool>
  </property>
  <widget class="QWidget" name="displayWidget" native="true">
   <property name="geometry">
    <rect>
     <x>260</x>
     <y>10</y>
     <width>61</width>
     <height>181</height>
    </rect>
   </property>
   <property name="autoFillBackground">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLabel" name="label_5">
   <property name="geometry">
    <rect>
     <x>260</x>
     <y>190</y>
     <width>71</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Hex</string>
   </property>
  </widget>
  <widget class="QSpinBox" name="redSpinBox">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>200</y>
     <width>42</width>
     <height>22</height>
    </rect>
   </property>
   <property name="maximum">
    <number>255</number>
   </property>
  </widget>
  <widget class="QSpinBox" name="greenSpinBox">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>200</y>
     <width>42</width>
     <height>22</height>
    </rect>
   </property>
   <property name="maximum">
    <number>255</number>
   </property>
  </widget>
  <widget class="QSpinBox" name="blueSpinBox">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>200</y>
     <width>42</width>
     <height>22</height>
    </rect>
   </property>
   <property name="maximum">
    <number>255</number>
   </property>
  </widget>
  <widget class="QSpinBox" name="opacitySpinBox">
   <property name="geometry">
    <rect>
     <x>190</x>
     <y>200</y>
     <width>42</width>
     <height>22</height>
    </rect>
   </property>
   <property name="maximum">
    <number>255</number>
   </property>
  </widget>
  <widget class="QWidget" name="">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>10</y>
     <width>221</width>
     <height>16</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <widget class="QLabel" name="label">
      <property name="text">
       <string>R</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="label_2">
      <property name="text">
       <string>G</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="label_3">
      <property name="text">
       <string>B</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="label_4">
      <property name="baseSize">
       <size>
        <width>0</width>
        <height>0</height>
       </size>
      </property>
      <property name="text">
       <string>A</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="">
   <property name="geometry">
    <rect>
     <x>-10</x>
     <y>30</y>
     <width>251</width>
     <height>161</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout_2">
    <property name="spacing">
     <number>8</number>
    </property>
    <item>
     <widget class="QSlider" name="redSlider">
      <property name="baseSize">
       <size>
        <width>64</width>
        <height>0</height>
       </size>
      </property>
      <property name="maximum">
       <number>255</number>
      </property>
      <property name="orientation">
       <enum>Qt::Vertical</enum>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QSlider" name="greenSlider">
      <property name="baseSize">
       <size>
        <width>64</width>
        <height>0</height>
       </size>
      </property>
      <property name="maximum">
       <number>255</number>
      </property>
      <property name="orientation">
       <enum>Qt::Vertical</enum>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QSlider" name="blueSlider">
      <property name="maximum">
       <number>255</number>
      </property>
      <property name="orientation">
       <enum>Qt::Vertical</enum>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QSlider" name="opacitySlider">
      <property name="maximum">
       <number>255</number>
      </property>
      <property name="orientation">
       <enum>Qt::Vertical</enum>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections>
  <connection>
   <sender>redSlider</sender>
   <signal>valueChanged(int)</signal>
   <receiver>redSpinBox</receiver>
   <slot>setValue(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>28</x>
     <y>118</y>
    </hint>
    <hint type="destinationlabel">
     <x>34</x>
     <y>204</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>redSpinBox</sender>
   <signal>valueChanged(int)</signal>
   <receiver>redSlider</receiver>
   <slot>setValue(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>15</x>
     <y>209</y>
    </hint>
    <hint type="destinationlabel">
     <x>21</x>
     <y>139</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>greenSlider</sender>
   <signal>valueChanged(int)</signal>
   <receiver>greenSpinBox</receiver>
   <slot>setValue(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>86</x>
     <y>97</y>
    </hint>
    <hint type="destinationlabel">
     <x>90</x>
     <y>212</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>greenSpinBox</sender>
   <signal>valueChanged(int)</signal>
   <receiver>greenSlider</receiver>
   <slot>setValue(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>77</x>
     <y>214</y>
    </hint>
    <hint type="destinationlabel">
     <x>77</x>
     <y>162</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>blueSlider</sender>
   <signal>valueChanged(int)</signal>
   <receiver>blueSpinBox</receiver>
   <slot>setValue(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>140</x>
     <y>96</y>
    </hint>
    <hint type="destinationlabel">
     <x>159</x>
     <y>210</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>blueSpinBox</sender>
   <signal>valueChanged(int)</signal>
   <receiver>blueSlider</receiver>
   <slot>setValue(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>137</x>
     <y>217</y>
    </hint>
    <hint type="destinationlabel">
     <x>138</x>
     <y>153</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>opacitySlider</sender>
   <signal>valueChanged(int)</signal>
   <receiver>opacitySpinBox</receiver>
   <slot>setValue(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>204</x>
     <y>134</y>
    </hint>
    <hint type="destinationlabel">
     <x>213</x>
     <y>206</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>opacitySpinBox</sender>
   <signal>valueChanged(int)</signal>
   <receiver>opacitySlider</receiver>
   <slot>setValue(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>193</x>
     <y>215</y>
    </hint>
    <hint type="destinationlabel">
     <x>198</x>
     <y>162</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
